### ISSUE TITLE
Project governance

### ISSUE THREAD

**Issue by celestehorgan** [inclusivenaming/org#5]

**Comment by ddavisjones** [comment]
@celestehorgan  if the goal of this issue is to create categories that would help contributors find their focus areas so that they can contribute more effectively I'd like to help with this. If it's something else, please add it to the description.

**Comment by edwarnicke** [comment]
@ddavisjones Shall we start brainstorming the problem statement here? :)

**Comment by ddavisjones** [comment]
@edwarnicke Sure!

**Comment by edwarnicke** [comment]
@ddavisjones my initial recollection of the problems we are trying to solve is:

1.  How to hold and protect trademarks
2. How to fund and pay cash expenses (things like domain registration fees etc)
3. How to support people expenses (like a full time PM for INI)

What other problems do we need to solve?

**Comment by ddavisjones** [comment]
Framing the Opportunity and  Problem Statement 

The Inclusive naming initiative was formed in November 2020 to accelerate the consistent, identification and replacement of non-inclusive terms used in IT by companies and projects. 
The premise was twofold  : 

1. There are enough learnings and insights from companies and projects who have already embarked on this journey to help those who are just starting.
2. Even those who are well on their way to making these language changes have open questions and issues that others in the Inclusive naming community have addressed 
3. Open questions that no one has been able to address, are more likely to be answered by diverse groups working on the question together than any one individual company or project,

The challenge of the inclusive naming working group is to create a collaboration model with the right tools and governance that strikes the right balance between structure and openness to  help the community achieve our collective goals

**Comment by edwarnicke** [comment]
@ddavisjones Some slight tweaks on your language:

Framing the Opportunity and Problem Statement
 
The Inclusive naming initiative was formed in November 2020 to accelerate the consistent, identification and replacement of non-inclusive terms used in IT by companies and projects.
The premise :
 
1. There are enough learnings and insights from **organizations,** companies **,** ~~and~~ projects **, and standards bodies**, who have already embarked on this journey to help those who are just starting.
2. Even those who are well on their way to making these language changes have open questions and issues that others in the Inclusive naming community have addressed
3. Open questions that no one has been able to address, are more likely to be answered by diverse groups working on the question together than any one individual company or project.
4. **Consistency requires collaboration.**
 
The challenge of the inclusive naming working group is to create a collaboration model with the right tools and governance that strikes the right balance between structure and openness to help the community achieve our collective goals

**Comment by ddavisjones** [comment]
@edwarnicke I like your tweaks.
 
 Here are my thoughts on what we need to append to the draft above  

An effective Collaboration model depends on 

1. A charter that outlines the expected contributions of all participants and leaders ,estimated time per week,  the way we would like to work, pull issues from a Kanban, success criteria we set for ourselves, how we gauge progress against those criteria 
2. People to do some level of coordination and project management 
3. People to manage our communications ( onboarding, press, accuracy and cohesion of content on our sites, requests for information)
4. People to do user experience research and retrospectives with members  to validate that what we have established works for them
5. Tools and infrastructure to support the work we do 
7. Funding and budget management for items that come with a cost ( since not all of this can be achieved through volunteers and donations)

**Comment by edwarnicke** [comment]
> @edwarnicke I like your tweaks.
> 
> Here are my thoughts on what we need to append to the draft above . what do you think?
> 
> An effective Collaboration model depends on
> 
> 1. A charter that outlines the expected contributions of all participants and leaders ,estimated time per week,  the way we would like to work, pull issues from a Kanban, success criteria we set for ourselves, how we gauge progress against those criteria

This feels a bit too detailed for me.  Usually, governance docs layout basic principles of organization, resourcing, and decision making... typically not tool choices etc.  Typically the reason for that is to try to get stable governance that changes infrequently, while allowing relatively permissionless innovation on questions of tools etc.

> 2. People to do some level of coordination and project management
> 3. People to manage our communications ( onboarding, press, accuracy and cohesion of content on our sites, requests for information)
> 4. People to do user experience research and retrospectives with members  to validate that what we have established works for them

I'd love to hear more about your thoughts on "People to do ..." with relation to governance :)  

> 5. Tools and infrastructure to support the work we do
> 6. Funding and budget management for items that come with a cost ( since not all of this can be achieved through volunteers and donations)

I'm also continuing to mull your layout of an effective collaboration model.  You come from a fascinatingly different perspective to it that I do (which, amusingly, makes us ideal collaborators on this)... trying to sync my brain more into your perspective :)

**Comment by ddavisjones** [comment]
@edwarnicke  Good point on too much detail in the definition of an effective collaboration model,
My primary reason for including it at all was to answer the question why governance?

Here’s my slimmed down version

An effective collaboration model optimizes the time  participants invest in value-added activities to achieve desired outcomes.
A governance structure establishes the  principles, policies, resources and decision criteria needed to sustain and protect the interests of the collaboration. 

This includes 

1. How to hold and protect trademarks
2. How to fund operational and infrastructure expenses such as project management, PR, network domain registration etc 
3. Participation expectations of members 


 
Your thoughts?
 




.

**Comment by edwarnicke** [comment]
> @edwarnicke Good point on too much detail in the definition of an effective collaboration model,
> My primary reason for including it at all was to answer the question why governance?
> 
> Here’s my slimmed down version
> 
> An effective collaboration model optimizes the time participants invest in value-added activities to achieve desired outcomes.

I like this line a lot :)
> A governance structure establishes the principles, policies, resources and decision criteria needed to sustain and protect the interests of the collaboration.
> 
> This includes
> 
> 1. How to hold and protect trademarks
> 2. How to fund operational and infrastructure expenses such as project management, PR, network domain registration etc
> 3. Participation expectations of members

I like this quite a lot as a starting place :)

**Comment by ddavisjones** [comment]
Framing the Governance Opportunity and Problem Statement

The Inclusive naming initiative was formed in November 2020 to accelerate the consistent, identification and replacement of non-inclusive terms used in IT by companies and projects.
The premise :

1. There are enough learnings and insights from organizations, companies , projects and standards bodies, who have already embarked on this journey to help those who are just starting.
2. Even those who are well on their way to making these language changes have open questions and issues that others in the Inclusive naming community have addressed
3. Open questions that no one has been able to address, are more likely to be answered by diverse groups working on the question together than any one individual company or project.
4. Consistency requires collaboration.


The challenge of the inclusive naming working group is to create a collaboration model with the tools and governance to strikes the right balance between structure and openness to help the community achieve our collective goals

An effective collaboration model optimizes the time participants invest in value-added activities to achieve desired outcomes.

A governance structure establishes the principles, policies, resources and decision criteria needed to sustain and protect the interests of the collaboration.

This includes

1.  How to hold and protect trademarks
2. How to fund operational and infrastructure expenses such as project management, PR, network domain registration etc
3. Participation expectations of members

**Comment by ddavisjones** [comment]
@edwarnicke  pulled it together in the posting above. does it work? 
Or is this statement unnecessary?
The challenge of the inclusive naming working group is to create a collaboration model with the tools and governance to strikes the right balance between structure and openness to help the community achieve our collective goals

Also, before we send it out for comments and edits should we include a statement at the end  like the following

Without a governance structure we risk cessation of progress due to churn, misdirection of priorities, an underfunded support model, burnout of some contributors and underutilization of others.

**Comment by geekygirldawn** [comment]
I'm a fan of putting this under a neutral foundation sooner, rather than later, with a preference for the Linux Foundation. The foundation can help us with implementing the governance structure.

**Comment by edwarnicke** [comment]
From the INI meeting today, we have the option of using a hackmd for drafting as well: https://hackmd.io/@inclusive-naming/BkVmwn07u/edit

**Comment by edwarnicke** [comment]
@geekygirldawn 
> I'm a fan of putting this under a neutral foundation sooner, rather than later, with a preference for the Linux Foundation. The foundation can help us with implementing the governance structure.

I'd love to understand *which* aspects of being under a 'neutral foundation' are most attractive to you, and what is driving your preference for the Linux Foundation :)

We've been trying so far to break down the problem before going to the solution... but proposed solutions such as yours usually shorthand *really* important problem aspects.  I'd love to capture them :)

**Comment by geekygirldawn** [comment]
@edwarnicke Similarly, I've also been doing this open source thing for 20+ years, and I work on governance quite a bit as an active participant in the CNCF Governance WG and as part of my role at VMware.

I've created a non-profit from scratch, which takes way too much time away from getting the real work done, so I'm a big fan of joining existing foundations. 

Things that are important to me:
- neutral governance, not controlled by any one company.
- a legal structure to own assets and in case something goes wrong.
- an easy way to pay for expenses to avoid individuals paying for things.
- access to other resources, like program managers.

The LF because:
- The LF is already donating time from several people, like Celeste and Priyanka, and they have expressed an interest in hosting this project.
- I'm part of a bunch of LF projects and they make things pretty easy.
- The downside is that they are a bit expensive, but this is important enough that we can probably cover it with sponsorships. 

This is sketchy and really incomplete, mostly because I don't have a ton of time to document this right now, but I'll try to think more about this and fill in the gaps.

**Comment by LarryKunz** [comment]
A topic that came up in yesterday's biweekly call: In addition to considering how to hold and protect trademarks, we should consider copyrights as well. 

Some of us on the call expressed a preference for having contributors retain copyrights for the material they contribute, rather than having INI claim copyrights. However, I'd like to hear everyone's thoughts.

Whatever we decide, it should be codified in the governance processes.

**Comment by edwarnicke** [comment]
> From the INI meeting today, we have the option of using a hackmd for drafting as well: [https://hackmd.io/@inclusive-naming/BkVmwn07u/edit ](https://hackmd.io/@inclusive-naming/BkVmwn07u/edit%E2%80%A9)

@justaugustus This link apparently isn't working for me... I perhaps transliterated it incorrectly.  Could you help?

**Comment by edwarnicke** [comment]
@geekygirldawn 
> * making it easier for companies to engage with the project.

Could you say more about this?  I'm acutely sensitive to the fact that different folks (companies being folks in this sentence) require different interfaces.  The simple example that comes to mind for me around companies is that they have particular ways of moving money around (the PO/approve vendor model) that require an entity on the other side of that which meshes with those methods.  Since many companies already have a 'billing relationship' with LF, it eases the process of donations.

Where there other sorts of 'engaging with the project' you had in mind?

**Comment by edwarnicke** [comment]
@ddavisjones - 
> @edwarnicke pulled it together in the posting above. does it work?
> Or is this statement unnecessary?

 I feel like the statement is necessary to the *process* of figuring out a governance.  I have no idea if it will be necessary in the end... that's the adventure :)

> The challenge of the inclusive naming working group is to create a collaboration model with the tools and governance to strikes the right balance between structure and openness to help the community achieve our collective goals
> 

YES!  I would also suggest one that preserves the organic energy that's come together in pursuit of the goal.  One of the things I'd like to avoid is a 'stop the world' governance scramble.  Its easy to get lost in the muck of to much governance to early to the detriment of the actual goals :)  I think we can avoid that :)

> Also, before we send it out for comments and edits should we include a statement at the end like the following
> 
> Without a governance structure we risk cessation of progress due to churn, misdirection of priorities, an underfunded support model, burnout of some contributors and underutilization of others.

I have a strong bias towards the positive, how about:

> A governance structure enables smooth progress, clarity of priorities, a funding model to support INI needs, and productive sustainable engagement of participants. 

I think that captures your concern in a positive tone, does it feel good to you?

**Comment by edwarnicke** [comment]
@ddavisjones @geekygirldawn - below is some synthesis and some of my tentative thoughts, additional input in all forms super welcome (especially on wording... @ddavisjones your written english has an elegance mine often lacks):

***************
### Preamble
The Inclusive Naming Initiative (INI) was formed to accelerate the consistent, identification and replacement of non-inclusive terms used in IT.
The premise :

1. Learnings and insights from organizations, companies , projects and standards bodies, who have already embarked on this  journey to help those who are just starting.
2. Even those who are well on their way to making these language changes have open questions and issues that others in the Inclusive naming community have addressed
3. Open questions that no one has been able to address, are more likely to be answered by diverse groups working on the question together than any one individual company or project.
4. Consistency requires collaboration.
The challenge of the inclusive naming working group is to create a collaboration model with the tools and governance to strikes the right balance between structure and openness to help the community achieve our collective goals

An effective collaboration model optimizes the time participants invest in value-added activities to achieve desired outcomes.

A governance structure establishes the principles, policies, resources and decision criteria needed to sustain and protect the interests of the collaboration.

A governance structure enables smooth progress, clarity of priorities, a funding model to support INI needs, and productive sustainable engagement of participants.

### Organizational Structure
INI has already spontaneously organized into workstreams:
* ws-admin
* ws-community-and-tooling
* ws-company
* ws-language
* ws-marketing
* ws-open-source
* ws-standards
each with their own leads and participants.

The workstream leads meet together regularly to coordinate and collaborate.   De facto the governance of the INI consists of the workstreams and the workstream leads group providing coordination.

The workstreams lead group should evolve to being the Board of the INI project, retaining its representation of each workstream.  The representative from each workstream to the board should be by default the lead for that workstream.  Each workstream may elect to send a representative who is not the workstream lead at their discretion.

The Board may also choose to extend itself with representatives for other constituencies, for example:
1.  Some number (1 or 2?) representatives of companies providing substantial material support, whether financial, human resources, or significant workproduct.
2. Some number (1 or 2?) representatives of standards bodies that have made a substantial commitment to collaboration with INI.

The Board may also choose to appoint non-voting advisors as it sees fit.

Board meetings should be default public, with the understanding that the Board may at times need to meet privately to discuss more sensitive matters.

The purpose of the Board is to facilitate collaboration and support of the INI community.   It is not to direct the INI community.  As such it may:
1. Facilitate collaboration
2. Mediate disputes
3. Raise funds to support the INIs needs

### Funding Model

Some activities require funding.  Examples include:
1. Holding and protecting trademarks
2. Funding services provided by vendors to INI such as:
    * network domain registration and other technical infrastructure
    * PR services
    * Program management services
    * Events

The Board should seek first and foremost to seek donations of services where possible.  Examples thus far include the gracious donation of [1password](https://github.com/1Password/1password-teams-open-source/pull/312).   Donations should be publicly acknowledge in a manner to be decided by the board.

Some needs may best be met by pooling of money from supporting organizations and individuals.  For those purposes, the Board should facilitate means to receive and hold funds for those purposes, bearing in mind the impact of mechanics of receipt of funds on the ability of donors to donate.  Examples: companies need to be able to engage at the level of POs, individuals more likely at the level of credit cards, etc.  Donations should be publicly acknowledge in a manner to be decided by the board.

Ideally, funding should be kept targeted and lean.

**Comment by geekygirldawn** [comment]
> @geekygirldawn
> 
> > * making it easier for companies to engage with the project.
> 
> Could you say more about this? I'm acutely sensitive to the fact that different folks (companies being folks in this sentence) require different interfaces. The simple example that comes to mind for me around companies is that they have particular ways of moving money around (the PO/approve vendor model) that require an entity on the other side of that which meshes with those methods. Since many companies already have a 'billing relationship' with LF, it eases the process of donations.
> 
> Where there other sorts of 'engaging with the project' you had in mind?

The funding one is the biggest. Most large companies make it nearly impossible to give projects money if they aren't already on the approved list, and the LF is already on that list for most of us. But, contribution can also be a barrier without a vendor-neutral foundation. I'm much more likely to contribute to a project if I can rest assured that we're all contributing on a level playing field and that no single vendor can somehow hijack the project.

**Comment by edwarnicke** [comment]
> @edwarnicke The funding one is the biggest. Most large companies make it nearly impossible to give projects money if they aren't already on the approved list, and the LF is already on that list for most of us. But, contribution can also be a barrier without a vendor-neutral foundation. I'm much more likely to contribute to a project if I can rest assured that we're all contributing on a level playing field and that no single vendor can somehow hijack the project.

Got it... and yes, I've experienced the pain of bringing new 'vendors' on at a big corp... its better to deal with one that is already on that list :)

**Comment by richsalz** [comment]
>I'm much more likely to contribute to a project if I can rest assured that we're all contributing on a level playing field and that no single vendor can somehow hijack the project.

If "you" means an individual, than perhaps setting up "GitHub sponsors" is something to pursue.

If "you" means the company you work for, then yeah, it can be hard.  Not only getting on the approved list, but handling 501c3 stuff, etc.  LF takes a percentage, but it's worth it in my opinion,

**Comment by geekygirldawn** [comment]
> > I'm much more likely to contribute to a project if I can rest assured that we're all contributing on a level playing field and that no single vendor can somehow hijack the project.
> 
> If "you" means an individual, than perhaps setting up "GitHub sponsors" is something to pursue.
> 
> If "you" means the company you work for, then yeah, it can be hard. Not only getting on the approved list, but handling 501c3 stuff, etc. LF takes a percentage, but it's worth it in my opinion,

This is a separate point about contribution from the standpoint of participation. This isn't about the money, it's about knowing that when I invest time in something (either as an individual or my company) that we're on a level playing field and that no single company can take control of these contributions.

**Comment by edwarnicke** [comment]
> > I'm much more likely to contribute to a project if I can rest assured that we're all contributing on a level playing field and that no single vendor can somehow hijack the project.
> 
> If "you" means an individual, than perhaps setting up "GitHub sponsors" is something to pursue.
> 
> If "you" means the company you work for, then yeah, it can be hard. Not only getting on the approved list, but handling 501c3 stuff, etc. LF takes a percentage, but it's worth it in my opinion,

Exactly... mechanisms matter.  Companies have POs and approved lists.  Individuals tend to have credit cards.  Methods will differ.

**Comment by edwarnicke** [comment]
> This is a separate point about contribution from the standpoint of participation. This isn't about the money, it's about knowing that when I invest time in something (either as an individual or my company) that we're on a level playing field and that no single company can take control of these contributions.

@geekygirldawn my experience has been that the concern is, as you point out, more about avoiding company dominance in the *activity* of projects.  Often projects are working in areas of active commercial interest for the companies involved... and there's a legitimate concern about one company 'controlling' the interest in that area.

I also get a sense in many cases of a human psychology issue around 'turf'.  Most folks feel better working on an even playing field, where it feels like you are a first class stakeholder, rather than a guest on someone else's 'turf'.

My sense is the 'one company controlling an area of commercial interest' part of it is much much less here, because I perceive what we are doing here (possibly incorrectly) as not being an area of commercial interest.  My sense is that the 'wanting to feel like first class stakeholders' part of neutrality is incredibly crucial in INI.  Its part of why I think its important that we construct INI in a way that feels that way not just for corps that are used to working in Open Source @LF, or for Open Source projects... but also for standards bodies, non-LF Open Source folks, and other sorts of individuals and non-corp orgs that want to get involved.

I feel like we may need to take a broader view of 'neutrality' we have often taken coming from the Open Source perspective :)

**Comment by richsalz** [comment]
My experience is similar, @edwarnicke : it's more about dominating the activities, rather than the funding.
